From 8e5726201e885a4fb67b3be011c178d05296e17c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 27 Sep 2024 11:16:38 +0800
Subject: [PATCH 2/3] AOSCOS: fix(completions): do not assume RPM even if
 /etc/debian_version is not found

AOSC OS is not a Debian-derivative nor an RPM distribution.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 completions/bash/apt | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/completions/bash/apt b/completions/bash/apt
index c249f97b1..01b3c716e 100644
--- a/completions/bash/apt
+++ b/completions/bash/apt
@@ -177,14 +177,8 @@ _apt()
     if [[ -v command ]]; then
         case $command in
             remove|purge|autoremove|autopurge|why)
-                if [[ -f /etc/debian_version ]]; then
-                    # Debian system
-                    COMPREPLY=( $( \
-                        _xfunc dpkg _comp_dpkg_installed_packages $cur ) )
-                else
-                    # assume RPM based
-                    _xfunc rpm _rpm_installed_packages
-                fi
+                COMPREPLY=( $( \
+                    _xfunc dpkg _comp_dpkg_installed_packages $cur ) )
                 return 0
                 ;;
             show|list|download|changelog|depends|rdepends|why-not)
-- 
2.52.0

